bindings/pyroot_experimental/pythonizations/CMakeLists.txt

```diff
@@ -92,7 +92,13 @@ foreach(val RANGE ${how_many_pythons})
 
   set(libname ROOTPythonizations${python_under_version_string})
 
-  ROOT_LINKER_LIBRARY(${libname} ${cpp_sources} DEPENDENCIES Core Tree cppyy${python_under_version_string} CMAKENOEXPORT)
+  add_library(${libname} SHARED ${cpp_sources})
+  set_target_properties(${libname} PROPERTIES  ${ROOT_LIBRARY_PROPERTIES})
+  if(APPLE)
+    target_link_libraries(${libname} PUBLIC -Wl,-bind_at_load -Wl,-undefined -Wl,dynamic_lookup Core Tree cppyy${python_under_version_string})
+  elseif(NOT MSVC)
+    target_link_libraries(${libname} PUBLIC -Wl,--unresolved-symbols=ignore-all Core Tree cppyy${python_under_version_string})
```

Comment:
To be checked with @bellenot : what would be the flags we should use for Windows?

Reply:
FYI, experimental pyroot is currently not supported on Windows (it never was). I have several patches for making it (more or less) working

Reply:
Ok, but we can make a choice of what we put here, the options are:
1. We don't include a `target_link_libraries` for Windows, i.e. we don't generate libROOTPythonizations at all for Windows (this is the current state of the PR).
2. We do generate the library with the equivalent flags to the ones proposed by this PR (leaving unresolved symbols of libpython).
3. We generate the library in the "old style" (i.e. linking to libpython).

What do you think @bellenot ?

Reply:
Just disable experimental PyROOT on Windows for the time being, I'll take care of all this later